index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href='https://fonts.googleapis.com/css?family=Poppins'>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Document</title>
+</head>
+<body>
+    <!-- toolbar -->
+    <nav class="navbar">
+        <h2 class="logo">My Logo</h2>
+        <a class="icon">
+            <i class="fa fa-bars"></i>
+        </a>
+    </nav>
+
+    <!-- headline -->
+    <section class="header">  
+        <h1 class="intro"><b>I'm Martin Glad to see you!</b></h1>
+        <p class="presentation">I'm a software developer! I can help you build a product, feature a website. 
+            Look through some of my work and experience!
+            If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+        <div class="social">
+            <p class="connect">LET'S CONNECT!</p>
+            <ul class="social_links">
+                <li><a href="https://www.twitter.com"><img class="social_buttons" src="images/twitter-icon.jpg" alt="twitter"></a></li>
+                <li><a href="https://www.linkedin.com/in/mart%C3%ADn-ossa-228672135/"><img class="social_buttons" src="images/Linkedin icon.jpg" alt="Linkedin"></a></li>
+                <li><a href="https://www.medium.com"><img class="social_buttons" src="images/medium-icon.jpg" alt="medium"></a></li>
+                <li><a href="https://www.github.com/maov19"><img class="social_buttons" src="images/github-icon.jpg" alt="github"></a></li>
+                <li><a href="https://www.facebook.com"></a></li>
+            </ul>
+        </div>
```

Comment:
The spacing was corrected, as requested.
Furthermore, the Twitter handle wasn't included because it isn't in the Figma design. I included it now!
Please let me know if there is anything else I should work on.